benchmark/run-bm-docker.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
- It'd be nice to add a instruction to how to build docker image and run `run-bm-docker.sh` in README

Reply:
ok, let me add.